eras/shelley/test-suite/bench/Main.hs

```diff
@@ -40,11 +46,15 @@ import Cardano.Ledger.Shelley.LedgerState
     incrementalStakeDistr,
     updateStakeDistribution,
   )
-import Cardano.Ledger.Shelley.PParams (ShelleyPParamsHKD (..))
+-- import Cardano.Ledger.Shelley.PParams (ShelleyPParamsHKD (..))
```

Comment:
Yeah, this import can be removed. It was only imported to bring `HasField` instances into scope.

Reply:
:+1: